src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+Nodes that can propose blocks are only a _subset_ of the total nodes on the network. This category includes mining nodes under proof-of-work (PoW), and validator nodes under proof-of-stake (PoS). This category requires committing economic resources (such as GPU hash power in PoW, or staked ETH in PoS), in exchange for the ability to occasionally propose the next block and earn protocol rewards.
+
+The _rest_ of the nodes on the network are not required to commit _any_ economic resources, beyond simply needing a consumer grade computer with 1-2 TB free storage and an internet connection. These nodes do not propose blocks, but still serve a _critical_ role in securing the network, by holding all block proposers accountable. These nodes listen for new blocks, and verify their validity on arrival according to the chosen rules of network consensus. If the block is valid, the node will continue propagating it through the network, but importantly, if the block is _invalid_ for whatever reason, the node software will simply disregard it as invalid, and stop its propagation.
+
+Running a "non block-producing" node is not only _possible_ for anyone under either consensus mechanism (PoW _or_ PoS), it is _strongly encouraged_ for all users to run their own node if possible. This is _immensely valuable_ for the network, and also gives added benefits to any individual running their own software, such as improved security, privacy and censorship resistance.
+
+The ability for anyone to run their own node is _absolutely essential_ to maintaining the decentralization of the Ethereum network.
+
+[More on running your own node](/run-a-node/)
+
+</ExpandableCard>
+
+<ExpandableCard
+title="Misconception: &quot;The Merge will reduce gas fees.&quot;"
+contentPreview="False. The Merge is a change of consensus mechanism, not an expansion of network capacity, and will not result in lower gas fees.">
+Gas fees are a product of network demand relative to the capacity of the network. The Merge deprecates the use of proof-of-work, transitioning to proof-of-stake for consensus, but does not significantly change any parameters that directly influence network capacity or throughput.
+
+With a [rollup-centric roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698), efforts are being focused on scaling user activity at [layer 2](/layer-2/), while enabling layer 1 Mainnet as a secure decentralized settlement layer optimized for rollup data storage to help make rollup transactions exponentially cheaper. The transition to proof-of-stake is a critical precursor to realizing this. [More on gas and fees.](/developers/docs/gas/)
+</ExpandableCard>
+
+<ExpandableCard
+title="Misconception: &quot;Transactions will be noticeably faster after The Merge.&quot;"
+contentPreview="False. Though some slight changes exist, transaction speed will mostly remain the same on layer 1.">
```

Comment:
Should the context preview be clearer here? I think the crux of it is:

transaction speeds will improve slightly but not in a way that will be noticeable to end-users.